workflow/tests/base_results/results_hvac_sizing.csv

```diff
@@ -119,6 +119,7 @@ base-enclosure-overhangs.xml,32302.0,8597.0,7508.0,0.0,575.0,6409.0,0.0,0.0,2446
 base-enclosure-rooftypes.xml,32302.0,8597.0,7508.0,0.0,575.0,6409.0,0.0,0.0,2446.0,2171.0,4597.0,17964.0,5216.0,7127.0,0.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,2520.0,0.0,0.0,0.0,0.0,36000.0,0.0,795.0,24000.0,998.0
 base-enclosure-skylights-physical-properties.xml,34658.0,8659.0,7508.0,2294.0,575.0,6409.0,0.0,0.0,2446.0,2171.0,4597.0,22841.0,5299.0,7127.0,4794.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,2520.0,0.0,0.0,0.0,0.0,36000.0,0.0,795.0,24000.0,998.0
 base-enclosure-skylights-shading.xml,32946.0,8615.0,7508.0,626.0,575.0,6409.0,0.0,0.0,2446.0,2171.0,4597.0,18743.0,5228.0,7127.0,766.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,2520.0,0.0,0.0,0.0,0.0,36000.0,0.0,795.0,24000.0,998.0
+base-enclosure-skylights-storms.xml,33472.0,8629.0,7508.0,1138.0,575.0,6409.0,0.0,0.0,2446.0,2171.0,4597.0,21450.0,5262.0,7127.0,3440.0,207.0,265.0,0.0,0.0,0.0,2010.0,619.0,2520.0,0.0,0.0,0.0,0.0,36000.0,0.0,795.0,24000.0,998.0
```

Comment:
We need to update our HVAC autosizing algorithm to incorporate the presence of storms too. Please look for references to window/skylight ufactor and shgc in hvac_sizing.rb. Let me know if you need any help.